CMakeSettings.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "environments": [ { "BuildNumber": "5180" } ],
+  "environments": [ { "BuildNumber": "5182" } ],
```

Comment:
that should be incremented.
maybe it worth to perform new release, actually.

it was my fault that I did not update it, I had an idea to make some automation, but it did not come to mind